data-raw/process_data_set_alina.R

```diff
@@ -0,0 +1,90 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+library(readr)
+
+# dem_candidates ---------------------------------------------------------------
+dem_candidates <- read_csv("data-raw/primary-candidates-2018/dem_candidates.csv") %>%
+  clean_names() %>%
+  #change classes from character to factor and date to date
+  mutate(
+    state = as.factor(state),
+    district = as.factor(district),
```

Comment:
Could you split `district` into the following variables (and update the help file accordingly):

* `body`: a factor with levels `governor`, `house`, `senate`
* `district`: a numerical variable equal to the district number only when `body` equals `house`, `NA` otherwise

No mention of state necessary since we already have `state`.

That way users can easily consider different bodies of government easily. If you're stuck on how to do this, ask @beanumber

Reply:
You should be able to do this using `str_detect()`s embeded within a `case_when()` embedded within this `mutate()`

Reply:
Or an `ifelse()` inside a `mutate()` would work as well.

Reply:
Thanks, Albert! I was wondering how to factor the `district` variable. Will do. So `body` is a new variable? Do I keep the current `district` variable as a variable with a different name?

Reply:
Please take the original `district` variable and split it into two: `body` and `district`. Or you could make the latter `district_num` and then delete the original `district` variable. please be sure to:
* Update the help file accordingly
* Run all package checks before commit/pushing
thanks!